doc/sphinx/source/vp/theorycov/examples.rst

```diff
@@ -35,121 +34,100 @@ relevant c-factors.
 
 .. code-block::  yaml
    :linenos:
-   
+
    meta:
-      author: Rosalyn Pearson
-      keywords: [theory uncertainties, 3-point]
-      title: NLO 3-point variations for 5 process types - DIS CC, DIS NC, DY, Top, Jets
-    
-   metadata_group: nnpdf31_process
-    
+   author: Andrea Barontini
```

Comment:
The indentation here is inconsistent with https://github.com/NNPDF/nnpdf/blame/fix_fivetheories_issues/doc/sphinx/source/vp/theorycov/examples.rst#L228-L231